app/assets/javascripts/AngularConfig.js.coffee

```diff
@@ -581,7 +592,7 @@ angular.module('dahlia.controllers',['ngSanitize', 'angular-carousel', 'ngFileUp
 @dahlia.run [
   '$rootScope', '$state', '$window', '$translate', 'ShortFormApplicationService', 'AccountService', 'ShortFormNavigationService',
   ($rootScope, $state, $window, $translate, ShortFormApplicationService, AccountService, ShortFormNavigationService) ->
-    # check if user is logged in
+    # check if user is logged in on page load
     AccountService.validateUser()
```

Comment:
@chrisdolendo see here -- so the `validateOnPageLoad` is an option for ngTokenAuth, it will automatically check if there is a logged in user session on page load. However we do that manually ourselves on pageLoad via `AccountService.validateUser()` so that we can also do some custom stuff like call `ShortFormApplicationService.importUserData(Service.loggedInUser)` which will pull in the logged in user's data into the `Service.application` object, or anything else we might want to do.